float64[] s_ref_spline
float64[] s_ref_mpc
float64[] s_traj_mpc
float64[] kappa_ref_spline
float64[] kappa_ref_mpc
float64[] kappa_traj_mpc